<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgLighting</class>
 <widget class="QDialog" name="DlgLighting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>277</width>
    <height>231</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lighting Settings</string>
  </property>
  <widget class="QGroupBox" name="groupBoxLighting">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>221</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Lighting Settings</string>
   </property>
   <widget class="QLineEdit" name="editLightDirX">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="editLightDirY">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="editLightDirZ">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelLightDir">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Direction:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelLightAmb">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ambient:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelLightDif">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Diffuse:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelLightSpe">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Specular:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelLightShine">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Shine:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editLightShine">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLightAmb">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLightDif">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLightSpe">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>190</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLightDefaults">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Restore Defaults</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
